commands/embed.js

```diff
@@ -130,19 +130,19 @@ class embed {
 		if (!message.member.roles.cache.has(config.roles.admin) && !config.allowedEmbedCmdUsers.includes(message.member.id)) {
 			return message.channel.send('You do not have permission to use this command.');
 		}
-		message.channel.send('Welcome to Horace Embed Creator!');
+		await message.channel.send('Welcome to Horace Embed Creator!');
 
-		let channel;
-		let title;
-		let url;
-		let description;
-		let subtitle;
-		let body;
-		let colour;
-		let imageLink;
 		let embedMessage;
 
 		try {
+			let channel;
```

Comment:
Sorry, I could have been more clear with this haha
Because they're all in the same scope now, you can just 
`let channel = await this.getChannel(message);`
`let title = await this.getTitle(message);`
`...`